README.rst

```diff
@@ -16,6 +9,46 @@ Purpose
 
 Redis Streams implementation for the Open edX event bus.
 
+Overview
+********
+This package implements an event bus for Open EdX using Redis streams.
+
+The event bus acts as a broker between services publishing events and other services that consume these events.
+
+This package contains both the publishing code, which processes events into messages to send to the stream, and the consumer code,
+which polls the stream using a `while True` loop in order to turn messages back into event data to be emitted.
```

Comment:
Can you clarify what "turn messages back into event data to be emitted" means here for people unfamiliar with our message bus implementation? I think we should also make clear that the actual event handling code for doing something with the messages lives elsewhere.